climate.py

```diff
@@ -9,6 +9,7 @@
 from custom_components.smartthinq import (
     CONF_LANGUAGE, KEY_DEPRECATED_COUNTRY,
     KEY_DEPRECATED_LANGUAGE, KEY_DEPRECATED_REFRESH_TOKEN)
+from wideq import ACHSwingMode, ACVSwingMode
```

Comment:
This is somewhat tricky but the rest of this module avoids `import wideq` at the top of the module so that we don't crash when trying to load the module, which gives HA a chance to install the dependency. Did something perhaps change so that's not necessary? If not, we may need to defer the import.

Reply:
Well, I need the definition of ACHSwingMode and ACVSwingMode in the SWING_MODES dict in the top. How would such defer look like? Never done that before.

Reply:
Aha, that's tricky. The basic answer is to wrap these in a function, so the `import` only happens when the data is actually _used_ rather when the module is first loaded. Does that make sense?

Reply:
Like 64f7563?

Reply:
That looks right to me! If it's working for you, let's merge this in.